jobs/pull-test-infra-bazel.sh

```diff
@@ -20,6 +20,7 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+pip install -r experiment/requirements.txt
```

Comment:
Rather than doing this, add an external workspace rule. We're actively trying to remove this file.